share/revng/test/tests/daemon/test.py

```diff
@@ -77,6 +77,7 @@ async def response_trace(session, trace_config_ctx, params: TraceRequestEndParam
 @fixture
 async def client(pytestconfig: Config, request) -> AsyncGenerator[AsyncClientSession, None]:
     temp_dir = TemporaryDirectory()
+    log_file = TemporaryFile("wb+", suffix="revng-daemon-test-log-")
```

Comment:
Why not use a `using` statement?

Reply:
* Everything under should be indented in the `with`
* `Temporary*` classes delete the resource either via `with` but also on destruction, so once `log_file` goes out of scope it will be deleted